data/human/post-war reactions.txt

```diff
@@ -0,0 +1,260 @@
+			choice
+				`	"Break it up, kids."`
+				`	(Ignore them and keep walking.)`
+					decline
+			`	The three attackers turn around, looking furious. They eye your gun and run away. "Thanks. I'm Gus," says the fourth boy, as he slowly rises to his feet. He looks about twelve years old.`
+			`	It takes Gus a few seconds to recover. He looks worried and asks, "Can you walk me to the repair shop? I'm already late for work, and I'm afraid they'll attack me again."`
+			label questions
+			choice
+				`	"Why were those boys attacking you?"`
+					goto attacking
+					to display
+						not "War Reactions: Syndicate Orphan: attacking"
+				`	"Aren't you a little young to work in a repair shop?"`
+					goto young
+				`	"I'm in a hurry. You're on your own."`
+					decline
+			label attacking
+			action
+				set "War Reactions: Syndicate Orphan: attacking"
+			`	"My sister is a Private, Third Class, in the Syndicate's Ancillary Security Service." He grins proudly, clearly having practiced that delivery. "She monitors the Syndicate workplaces and she turned those boys in for smoking on the job last week. I guess they finally got a break for the repair shop and came after me since they can't hurt my sister."`
+				goto questions
+			label young
+			`	As you start walking, he answers. "No, I'm an orphan." He sees your confusion and explains. "The Syndicate lets some of us orphans work in industry. In return they give us better schooling, nicer housing, and doctor's visits." He puffs himself up. "Only the cleverest of us get to do it. All of the jobs for orphans are safe too; we don't use big dangerous equipment. Just ordinary stuff, like hammers, soldering irons, or sewing machines.`
+			`	"Wait a minute!" He suddenly looks at you with wide eyes. "You're a captain, right? Were you in the big fights? I was on Maker when I saw some Pug ships fighting our ships on our school's big edu-screen."`
+			choice
+				`	"Yes, I'm Captain <first> <last>."`
+					goto me
+				`	"I blew up my share of Pug ships."`
+					goto blew
+				`	"Actually, I led the battle."`
+			`	"Oh, hold on," he says, pulling out a durable-looking data pad labeled 'Syndicate Childcare Device'. After some typing and swiping, he declares, "You're Captain <first> <last>! Wow! It's an honor to meet you, sir. Can I shake your hand?" He puts out his hand, and you shake it.`
+				goto next
+			label me
+			`	"Oh, hold on," he says, pulling out a durable-looking data pad labeled 'Syndicate Childcare Device'. After some typing and swiping, he declares, "You led the attack! Wow! It's an honor to meet you, sir. Can I shake your hand?" He puts out his hand, and you shake it.`
+				goto next
+			label blew
+			`	"I bet you did! I wish I was on one of our warships, bravely defending us all. Blowing up Pugs, getting paid. It sounds amazing!"`
```

Comment:
Yeah, just don't hope to be on my flagship: boarding Pug ships is dangerous, especially for children.

Reply:
He is too young to have considered those consequences, and he likely isn't aware of the Pug fondness for bioweapons.